### ISSUE TITLE
The broadcast receiver is registered incorrectly

### ISSUE THREAD

**Issue by AhmedGaberAbass** [jeancsanchez/JCPlayer#145]
I got this error from Google Play, Is there a solution to it?

One problem has been detected
We've spotted some common issues that prevent your app from being submitted for review. Please resolve these issues before submitting your changes for review.
Technical quality
The broadcast receiver is registered incorrectly
Applications targeting Android version 13 or later must specify export behavior when registerReceiver() is executed. If the export behavior is not specified, your application may crash.

Your app targets Android version 13 or later and registers non-system broadcast receivers without specifying export behavior in the following places:

Please update your application to specify export behavior for registered broadcast receivers.

**Comment by jeancsanchez** [comment]
Hi @AhmedGaberAbass 

I reviewed this issue and I couldn't  find anything related to it. The `JcPlayerNotificationReceiver`  already is registered in Manifest file with the `exported` property  specified:

<img width="632" alt="Screenshot 2024-07-10 at 20 18 28" src="https://github.com/jeancsanchez/JcPlayer/assets/11152015/10aa2b42-1feb-43e2-b3e9-dcee21eb5354">


I wonder if there is something in your application that is registering a [context registered receivers](https://developer.android.com/guide/components/broadcasts#context-registered-receivers) somewhere.

**Comment by dzmitrykavalioum** [comment]
HI @AhmedGaberAbass 
was this issue solved ?

i observe the same issue at my side and what is interesting - google console didn't provide places where problem was found.

**Comment by AhmedGaberAbass** [comment]
Hi, how are you, I hope you are fine
- No, the problem is not solved yet, but I think the problem with the
permission to access notifications in Android 13 and above is the reason.
- Is it possible to check this problem and let me know if you find a
solution?!
Regards

في الاثنين، ١٩ أغسطس ٢٠٢٤، ٥:٣٠ م Dzmitry Kavaliou ***@***.***>
كتب:

**Comment by dzmitrykavalioum** [comment]
Hi
Sorry for the delay

How do you register your BroadcastReceiver - could you show - how it was implemented at your side?

**Comment by dzmitrykavalioum** [comment]
I don't think that problem relates to permissions (in this case warning must contain permission error)

**Comment by AhmedGaberAbass** [comment]
Hi @jeancsanchez , @dzmitrykavalioum 
- I solved the problem by adding these lines in AndroidManifest
`  <receiver
            android:name="com.example.jean.jcplayer.service.notification.JcPlayerNotificationReceiver"
            android:enabled="true"
            android:exported="true"
            android:permission="android.permission.POST_NOTIFICATIONS"
            tools:replace="android:exported" />

    </application>`

**Comment by AhmedGaberAbass** [comment]
<receiver
            android:name="com.example.jean.jcplayer.service.notification.JcPlayerNotificationReceiver"
            android:enabled="true"
            android:exported="true"
            android:permission="android.permission.POST_NOTIFICATIONS"
            tools:replace="android:exported" />
